courses/api.py

```diff
@@ -776,48 +777,56 @@ def generate_course_run_certificates():
         return
 
     for run in course_runs:
-        edx_grade_user_iter = exception_logging_generator(
-            get_edx_grades_with_users(run)
-        )
-        created_grades_count, updated_grades_count, generated_certificates_count = (
-            0,
-            0,
-            0,
+        generate_course_run_certificates_for_course(run)
+
+
+def generate_course_run_certificates_for_course(run):
```

Comment:
It might be helpful to rename both `generate_course_run_certificates()` and `generate_course_run_certificates_for_course()` to more clearly distinguish what they do.  Can you also add in type hints for the parameter?

Reply:
I think the names are pretty accurate. What would be your suggestion?

Reply:
Yeah I'm not sure why I was originally thinking both should be renamed.  It might be helpful to rename `generate_course_run_certificates_for_course` to `generate_course_run_certificates_for_course_run` but that's nit-picking.  I would still add type hints for the `run` parameter.